x/clp/types/msgs.go

```diff
@@ -127,7 +127,7 @@ func (m MsgRemoveLiquidity) ValidateBasic() error {
 	if !(m.WBasisPoints.IsPositive()) || m.WBasisPoints.GT(sdk.NewInt(MaxWbasis)) {
 		return sdkerrors.Wrap(ErrInvalidWBasis, m.WBasisPoints.String())
 	}
-	if m.Asymmetry.GTE(sdk.NewInt(10000)) || m.Asymmetry.LTE(sdk.NewInt(-10000)) {
+	if m.Asymmetry.GT(sdk.NewInt(10000)) || m.Asymmetry.LT(sdk.NewInt(-10000)) {
```

Comment:
To confirm: will error if greater than 10,000 or less than -10000 ?

Reply:
Yes, but will accept == 10000 and == -10000